src/utils/flow.js

```diff
@@ -46,6 +46,21 @@ export function convertFlowObjectTypeAnnotation(
       };
     }
 
+    if (value.type === "ArrayTypeAnnotation" && value.elementType) {
+      const children = value.elementType.id && flowTypes[value.elementType.id.name]
+        ? flowTypes[value.elementType.id.name]
+        : value.elementType;
+
+      return {
+        ...obj,
+        [key]: {
+          type: "array",
+          nullable: property.optional,
+          children: children.properties ? convertFlowObjectTypeAnnotation(children, flowTypes) : null
```

Comment:
I don't know why `children.properties` typechecks with flow, it should really be `children.type === "ObjectTypeAnnotation"` as that ensures a `ObjectTypeAnnotation` is being passed to `ObjectTypeAnnotation`.